swagger.yaml

```diff
@@ -4290,13 +8238,78 @@ definitions:
         description: JSON-formatted string containing the usage information by feature name
         x-go-custom-tag: gorm:"type:text"
 
+  infraenv-create-params:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the InfraEnv.
+      proxy:
+        $ref: "#/definitions/proxy"
+      additional_ntp_sources:
+        type: string
+        description: A comma-separated list of NTP sources (name or IP) going to be added to all the agents.
+      ssh_authorized_key:
+        type: string
+        description: SSH public key for debugging the installation.
+      pull_secret:
+        type: string
+        description: The pull secret obtained from Red Hat OpenShift Cluster Manager at cloud.redhat.com/openshift/install/pull-secret.
+      static_network_config:
+        type: array
+        items:
+          $ref: '#/definitions/agent_static_network_config'
+      image_type:
+        $ref: '#/definitions/image_type'
+      ignition_config_override:
+        type: string
+        description: Json formatted string containing the user overrides for the initial ignition config
+      cluster_id:
+        type: string
+        format: uuid
+        description: If set, all agents that register will be associated with the specified cluster
 
-  image_info:
+  infra_env:
```

Comment:
what about openshift version for infra_env? it is used to determine the basic ISO

Reply:
Yep

Reply:
added